### DISCUSSION TITLE
Straempler Encoder directon

### DISCUSSION THREAD

**Discussion by bretdennison** [ctag-fh-kiel/ctag-straempler#22] [Q&A]
Is there a way to reverse the encoder direction?

**Comment by ctag-fh-kiel** [comment]
You would have to build your own firmware. If you configure the project with idf.py menuconfig, there is a configuration option to invert the encoder.
If you clone the repository on Github, you can use the Github action to build the firmware in the cloud. However, before doing that you would have to manually edit the sdkconfig.default file and comment out line 125 (CONFIG_SWAP_ENCODER_PINS=y). That would be the equivalent of doing it with idf.py menuconfig.
